src/components/ThreadEnvelope.vue

```diff
@@ -412,20 +419,26 @@ export default {
 			}
 		}
 	}
-	.app-content-list-item-star.icon-starred {
+	.app-content-list-item-star.favorite-icon-style {
 		display: inline-block;
 		position: absolute;
-		margin-top: -2px;
-		margin-left: 27px;
+		margin-top: -24px;
+		margin-left: 28px;
 		cursor: pointer;
+		&:hover {
+			opacity: .4;
```

Comment:
```suggestion
			opacity: .5;
```

To make it consistent with the important icon.